openmls/src/messages/proposals.rs

```diff
@@ -208,12 +208,12 @@ impl AddProposal {
     Debug, PartialEq, Clone, Serialize, Deserialize, TlsDeserialize, TlsSerialize, TlsSize,
 )]
 pub struct UpdateProposal {
-    pub(crate) key_package: KeyPackage,
+    pub key_package: KeyPackage,
```

Comment:
Why does the member need to be public? There's the `key_package` function below to get it.